src/deserialize/parse.rs

```diff
@@ -262,15 +262,15 @@ fn read_bool_or_nil(
         .next()
         .ok_or_else(|| Error::ParseEdn("Could not identify symbol index".to_string()))?
         .0;
+    let delimiters = [',', ']', '}', ')', ';'];
```

Comment:
constify

Reply:
Unless I'm missing something
both 
`const delimiters: [char; 5] = [',', ']', '}', ')', ';'];`
and 
`let delimiters = [',', ']', '}', ')', ';'];`
are [char; 5] and don't require an allocation. I find the latter easier to read and less manual type typing. Open to suggestions

Reply:
const outside the function call, as a global const:

const DELIMITERS: [CHAR;5] = [',', ']', '}', ')', ';'];

I don't think I have ever used const inside a function, but yeah, they would be +- the same thing